src/qtism/runtime/tests/AbstractSessionManager.php

```diff
@@ -195,8 +197,6 @@ protected function createRoute(AssessmentTest $test): Route
                         // Add to the last item of the selection the branch rules of the AssessmentSection/testPart
                         // on which the selection is applied... Only if the route contains something (empty assessmentSection edge case).
                         if ($route->count() > 0) {
-                            $route->getLastRouteItem()->addBranchRules($current->getBranchRules());
-
                             // Do the same as for branch rules for pre conditions, except that they must be
                             // attached on the first item of the route.
                             $route->getFirstRouteItem()->addPreConditions($current->getPreConditions());
```

Comment:
Maybe we also need to remove this one, wdyt?

Reply:
Yes, does not make sense anymore. Removed :)